src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -1207,5 +1207,15 @@ CHIP_ERROR ExtractPubkeyFromX509Cert(const ByteSpan & certificate, Crypto::P256P
     return CHIP_ERROR_NOT_IMPLEMENTED;
 }
 
+CHIP_ERROR ExtractAKIDFromX509Cert(const ByteSpan & certificate, MutableByteSpan & akid)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
+
+CHIP_ERROR ExtractVIDFromX509Cert(const ByteSpan & certificate, uint16_t & vid)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
```

Comment:
Please ensure arguments are ignored to avoid compile errors:

`(void)certificate`, etc